.github/workflows/ci.yml

```diff
@@ -232,13 +271,16 @@ jobs:
       - name: Build in JVM mode
         shell: bash
         run: |
-          MODULES_MAVEN_PARAM=""
           if [[ -n ${MODULES_ARG} ]]; then
             echo "Running modules: ${MODULES_ARG}"
-            MODULES_MAVEN_PARAM="-pl ${MODULES_ARG}"
+          mvn -B --no-transfer-progress -fae -s .github/quarkus-snapshots-mvn-settings.xml clean verify -Dall-modules -pl ${MODULES_ARG} -am -DexcludedGroups=long-running
+          fi
+          
+          # Long running tests
+          if [[ -n ${MODULES_LONG_RUNS_ARG} ]]; then
+            echo "Running modules: ${MODULES_LONG_RUNS_ARG}"
+            mvn -B --no-transfer-progress -fae -s .github/quarkus-snapshots-mvn-settings.xml verify -Dall-modules -pl ${MODULES_LONG_RUNS_ARG} -am
```

Comment:
Hey @gtroitsk , please patience, but I'll need to understand what you are trying to do. What I see:

- you detect long-running tests
- if they are among changed files, you run them

but if the same module (project) is changed, but not the long-running tests, how exactly do you know that long-running test wasn't affected by changes? I doubt that long-running tests are isolated, so if they can be affected by changed files around (like the application.properties that long-running test use), you don't want to run them?

Reply:
Script scans all changed files in module and if module contains long-running annotated file adds this module to `MODULES_LONG_RUNS_ARG` var. You can see printed modules for each category in first jobs from https://github.com/gtroitsk/quarkus-test-suite/actions/runs/12012099361/job/33482565059?pr=4

Reply:
You did not answer @gtroitsk . Please read my comment and answer why would you ignore module changes that can affect long-running test. Thanks

Reply:
@michalvavrik [This change](https://github.com/gtroitsk/quarkus-test-suite/pull/4/files#diff-48d661ee020bbd17564ba9adaa63e9f5291940c08d716a76385c3640f2d3943a) in module that has annotated test(but not changed even class) address situation that you mentioned, because I don't understand